.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Tracking Issue
+about: Track the progress of a specific task or feature
+title: '[TRACKING] Short summary of the task or feature'
+labels: 'C-tracking-issue'
+
+---
+### Description
+<!-- Provide a brief description of the task or feature being tracked！ -->
```

Comment:
Insert a blank line between the title and body of the section.

Reply:
Thanks for your reminding！